koiki/client.py

```diff
@@ -21,20 +22,30 @@ def __init__(self, order, auth_token=koiki.auth_token, logger=koiki.logger):
 
     def create_delivery(self):
         endpoint_req = CreateDelivery(self.order)
+        order = Order(self.order)
         req_body = self._authentication(endpoint_req)
 
         url = self._url(endpoint_req)
+        vendors = endpoint_req._vendors()
         self.logger.info('Koiki request to {}. body={}'.format(url, req_body))
 
         response = requests.post(url, json=req_body)
         response_body = json.loads(response.text)
 
-        if self._is_errored(response_body):
-            self._log(response.status_code, response.text, level='error')
-            return Error(response_body)
-        else:
-            self._log(response.status_code, self._masked_body(response_body))
-            return Delivery(response_body['envios'][0])
+        # TODO: check errors per Envio, not from general response,
+        # in case some are ok in a multiple vendor case.
+        deliveries = []
+        for n, d in enumerate(response_body['envios']):
```

Comment:
I would really prefer to use non-single-letter variables. It makes it quite hard to understand.